### PR TITLE
Feature/command

### PR DESCRIPTION
**Pull request by maxcnunes**
Add a command property specifying the query type. It is a base implementation for sqlectron/sqlectron-gui#134

### CODE REVIEW

File: spec/db.spec.js [hunk 1]
```diff
@@ -491,29 +491,93 @@ describe('db', () => {
...
                 expect(secondResult).to.have.property('fields').to.eql([]);
-                expect(secondResult).to.have.property('isSelect').to.eql(false);
                 expect(secondResult).to.have.property('rowCount').to.eql(undefined);
                 expect(secondResult).to.have.property('affectedRows').to.eql(1);
               }
             });
           });
+
+          describe('CREATE', () => {
+            describe('DATABSE', () => {
```
Comment (BornaP) [inline spec/db.spec.js]: Small typo, 'A' is missing. Same thing down below on line 551.

File: src/db/clients/mysql.js [hunk 2]
```diff
@@ -143,17 +144,18 @@ export function listTableTriggers(client, table) {
 }
 
 export function executeQuery(client, query) {
+  const commands = identifyCommands(query);
   return new Promise((resolve, reject) => {
     client.query(query, (err, data, fields) => {
       if (err && err.code === mysqlErrors.ER_EMPTY_QUERY) return resolve([]);
       if (err) return reject(_getRealError(client, err));
 
       if (!isMultipleQuery(fields)) {
```
Comment (BornaP) [inline src/db/clients/mysql.js]: Is it neccessary to have this check if we use `data.map` on resolve later?

**Comment by BornaP** [comment]
Nice work with sql-query-identifier! It will be a great feature :clap: :tada:

File: src/db/clients/mysql.js (hunk 2 above)
Comment (maxcnunes) [inline src/db/clients/mysql.js]: I guess not. I will try to test it out of this commit and check if the tests pass.

Comment (maxcnunes) [inline src/db/clients/mysql.js]: Yeah. That is necessary. With single query, fields and data will return inside another array.

**Comment by BornaP** [comment]
Since @maxcnunes has pushed changes related to this PR directly to master, and published new major on NPM as well, I'll close this pull request.

**Pull request closed without merging**